main.py

```diff
@@ -199,10 +200,10 @@ def run_search_interactive(kontaktdaten_path, check_delay):
     print()
     kontaktdaten = update_kontaktdaten_interactive(
         kontaktdaten, "search", kontaktdaten_path)
-    return run_search(kontaktdaten, check_delay)
+    return run_search(kontaktdaten, check_delay,telegram)
 
 
-def run_search(kontaktdaten, check_delay):
+def run_search(kontaktdaten, check_delay, telegram=None):
```

Comment:
`telegram` wird immer übergeben, oder? Wozu der Default?

```suggestion
def run_search(kontaktdaten, check_delay, telegram):
```

Reply:
War mir nicht sicher von wo genau das alles aufgerufen wird, mit dem default wäre es auf jeden Fall weiterhin gegangen

Reply:
Kannst ja das Repo durchsuchen. Außerhalb der `main.py` sollte das eigentlich nicht aufgerufen werden, da die `main.py` ja der Einstiegspunkt ist und wie sonst eine zyklische Abhängigkeit hätten.